src/dune_rules/merlin/merlin.ml

```diff
@@ -327,7 +327,15 @@ module Processed = struct
         |> Pp.concat_map ~sep:Pp.cut ~f:pp_one
         |> Pp.vbox
       in
-      Format.printf "%a%a@." Format.pp_set_margin 1000 Pp.to_fmt pp
+      let fmt =
```

Comment:
style remark: I'd expect `fmt` to be either a format string or a formatter (though `ppf` is preferred for that) but it's a formatting function applied to `pp`, so how about renaming it to `format_pp`?

Reply:
I know, normally this would be called `pp(_type)` but the conventions in `Pp` are a bit odd and the code calls the value to be printed `pp` already.

I started with renaming one the `pp` and the other one to `v` but then there was one place in the code where the conventions would be broken and it didn't seem worth it being inconsistent. I can name it `format_pp` to reduce confusion and convention-breaking.